lib/src/signed_video_tlv.c

```diff
@@ -902,6 +903,43 @@ tlv_find_tag(const uint8_t *tlv_data, size_t tlv_data_size, sv_tlv_tag_t tag, bo
   return NULL;
 }
 
+svi_rc
```

Comment:
This function is used to as a boolean, by checking if it returns SVI_OK. How about returning a bool directly?